### PR TITLE
[13.0][ADD] stock_inventory_valuation_show_remaining

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
When you trigger a report of inventory valuation the total quantity and the total value of the layers show. This is confusing as users expect is the actual value sitting in stock. This module modifies the report to show the remaining values.

### CODE REVIEW

**Comment by JordiMForgeFlow** [comment]
Code and functional review 👍🏼

**Review by JordiMForgeFlow — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]
Should the name be `stock_inventory_valuation_show_remaining` instead? 🤔

**Comment by AaronHForgeFlow** [comment]
yes, I think that is better name, I'll do that.

File: stock_inventory_valuation_show_remaining/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+When you trigger a report of inventory valuation the total quantity and the
+total value of the layers show. This is confusing as users expect is the
```
Comment (rousseldenis) [inline stock_inventory_valuation_show_remaining/readme/DESCRIPTION.rst]:
```suggestion
total value of the layers are displayed. This is confusing as users expect is the
```

Comment (AaronHForgeFlow) [reply stock_inventory_valuation_show_remaining/readme/DESCRIPTION.rst]: Thanks! applied that in the last commit.

**Review by LoisRForgeFlow — APPROVED** [review]

**Pull request merged**